server/src/test/java/io/crate/execution/ddl/tables/AddColumnTaskTest.java

```diff
@@ -326,4 +326,75 @@ public void test_raises_error_on_nested_arrays() throws Exception {
                 .hasMessageContaining("nested arrays are not supported");
         }
     }
+
+    @Test
+    public void test_stores_news_columns_with_assigned_oid() throws Exception {
```

Comment:
I miss the meaning of this test. Afaik this is not testing anything of the `AddColumnTask` beside returning newly added columns (as references). Testing that `Reference.toMapping` writes out OID's correctly should be tested at the related reference tests.

Reply:
this is basically testing that new `AddColumnResponse` returns what we need - only columns with "just now assigned oids".

we have the test for streaming response with the list-->we use AddColumnTask to get that list -> here we test that list in the response is correct.

assigning oids logic is done and tested in https://github.com/crate/crate/commit/f1b695a90a03ec8d0d2f4ce950867b6b432fe33c, here we don't change assignment logic, we just return some useful payload back to the caller.

Reply:
Ah now I understand it, you want to test that all references are collected and added to `addedColumns()`, that's what you mean with `stores`. Ok then please rename it to (it has nothing do to with oid's):
```suggestion
    public void test_collect_new_columns() throws Exception {
```